ariadne_codegen/client_generators/result_types.py

```diff
@@ -152,8 +155,14 @@ def _get_operation_type_name(self, definition: ExecutableDefinitionNode) -> str:
         raise NotSupported(f"Not supported operation type: {definition}")
 
     def generate(self) -> ast.Module:
-        module_body = cast(List[ast.stmt], self._imports) + cast(
-            List[ast.stmt], self._class_defs
+        model_rebuild_calls = [
+            generate_expr(generate_method_call(class_def.name, MODEL_REBUILD_METHOD))
+            for class_def in self._class_defs
```

Comment:
This list will have to be filtered for classes that actually need rebuilding.

Reply:
ok, thanks, now I know exactly what the problem is.